indexer/indexer.py

```diff
@@ -18,25 +18,37 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import os
+import re
 import gzip
 import logging
 import zipfile
 from lxml import etree as ET
 from xml.dom import minidom
 from distutils.version import LooseVersion
 
+#logging.basicConfig(level=logging.DEBUG)
+logging.basicConfig(level=logging.INFO)
+#logging.basicConfig(level=logging.WARN)
 
 def split_version(path):
-    return os.path.splitext(os.path.basename(path))[0].rsplit('-', 1)
+    result = os.path.splitext(os.path.basename(path))
```

Comment:
I would add a comment explaining what we're doing here and why. Especially the regex is not self explaining.